### ISSUE TITLE
link to documentation

### ISSUE THREAD

**Issue by vks** [VcDevel/Vc#71]
Currently it is not possible to find the documentation when reading the README. Ideally there should be a hint how to build it (doxygen) and a link to the build html docs. I think we can setup Travis to build the docs.

**Comment by vks** [comment]
Here is how to setup Travis to push the docs to Github pages: https://stackoverflow.com/questions/23277391/how-to-publish-to-github-pages-from-travis-ci

It is a bit tricky to setup in a secure way.

**Comment by mattkretz** [comment]
I agree that the link should appear in the README to show on the frontpage here on github. I'm not sure I want to go through the hassle of pushing from Travis, though. After all, I already have a working cron-job on the GSI Linux farm that publishes the documentation on https://web-docs.gsi.de/~mkretz/Vc-master/.

**Comment by vks** [comment]
Then maybe just add a link to the docs hosted at GSI?

**Comment by mattkretz** [comment]
Still somewhat undecided. Since there's a link in the wiki already. Maybe I should just link to the wiki page from the README. I'll play with it a bit and see what feels better.
